src/Integration.UnitTests/LocalServices/SolutionBindingInformationProviderTests.cs

```diff
@@ -82,15 +87,33 @@ public void SolutionBindingInformationProvider_GetUnboundProjects_SolutionBound_
         }
 
         [TestMethod]
-        public void SolutionBindingInformationProvider_GetUnboundProjects_ValidSolution_SolutionRuleSetIsMissing()
+        public void GetUnboundProjects_Connected_SolutionBound_EmptyFilteredProjects()
```

Comment:
For now, I've just duplicated the existing _legacy_ tests with the binding mode set to _connected_.
I considered try to remove some of the duplication in the test code, but decided to do that later.